tests/suites/test_suite_psa_crypto_driver_wrappers.data

```diff
@@ -39,3 +39,87 @@ generate_key:PSA_ERROR_NOT_SUPPORTED:"":PSA_SUCCESS
 
 generate_key through transparent driver: error
 generate_key:PSA_ERROR_GENERIC_ERROR:"":PSA_ERROR_GENERIC_ERROR
+
```

Comment:
Please add some negative testing as well as you did in #3501. We should have for each driver entry point at least one test where the driver entry point returns in error.